PendingReleaseNotes

```diff
@@ -17,6 +17,24 @@
   listed in 'osd class default list' requires a capability naming the class
   (e.g. 'allow class foo').
 
+* The 'rgw_rest_getusage_op_compat' config option allow you to dump the description of user stats
+  or not in s3 GetUsage API.This config is bool type and its default value is false.
```

Comment:
space after period, and `this config is` -> `this config option is of` , also `its default value is false` -> `defaults to false`